src/components/Movie/Movie.js

```diff
@@ -1,21 +1,44 @@
 import React, {Component} from 'react';
 import {Link} from 'react-router-dom';
-import { sortGenres } from '../../utils'
+// import { sortGenres } from '../../utils'
 import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faHeart } from '@fortawesome/free-solid-svg-icons'
+import { sortGenres } from '../../utilities/utils';
 
 class Movie extends Component {
-  componentDidMount() {
-    this.props.selectMovie(this.props.id)
-  }
-  render() {
-      const {backdrop_path, title, average_rating,
-          release_date, overview, genres = [], budget,
-            revenue, runtime, tagline} = this.props.movieInfo;
-      let genreTags = sortGenres(genres)
-      return (         
+    constructor(props) {
+      super(props)
+      this.state = {
+        message: ''
+      }
+    }
+    componentDidMount() {
+       this.props.selectMovie(this.props.id)
+    }
+
+    determineFavoriteUnfavorite() {
+      if (this.props.movieInfo.isFavorited && !this.state.message) {
+        this.props.unFavoriteMovie(this.props.id)
+        this.setState({message: 'Removed from Favorites'})
+        setTimeout(() => this.setState({message:''}), 3000);
+      } else if (!this.props.movieInfo.isFavorited && !this.state.message){
+        this.props.favoriteMovie(this.props.id)
+        this.setState({message: 'Added to Favorites'})
+        setTimeout(() => this.setState({message:''}), 3000);
+      }
+    }
+
+    render() {
+        const {backdrop_path, title, average_rating,
+            release_date, overview, genres = [], budget,
+             revenue, runtime, tagline} = this.props.movieInfo;
+       let genreTags = sortGenres(genres);
+       return (
         <article className='selected-movie'>
         <div className="img-container">
           <img src={backdrop_path} alt={title} />
+          <FontAwesomeIcon className={this.props.movieInfo.isFavorited? "favorite-button favorited" : "favorite-button"} icon={faHeart} size="3x" onClick={() => {this.determineFavoriteUnfavorite()}}/>
```

Comment:
I like how you were able to import font awesome into React, which is well.. awesome!  Lets look into moving the size attr into the CSS